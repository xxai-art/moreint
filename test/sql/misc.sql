SELECT - '5'::i8;
SELECT - '127'::i8;
SELECT - '-128'::i8;
