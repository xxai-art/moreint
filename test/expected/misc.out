SELECT - '5'::i8;
 ?column? 
----------
 -5
(1 row)

SELECT - '127'::i8;
 ?column? 
----------
 -127
(1 row)

SELECT - '-128'::i8;
ERROR:  integer out of range
